### PR TITLE
Update NLB Health check parameters to match docs, allow NLB un/healthy counts to be different

### LINKED ISSUE #732: elbv2 validation HealthCheckTimeoutSeconds and HealthCheckIntervalSeconds might be incorrect?
https://github.com/compose-x/ecs_composex/blob/9c3be5bf32fa5a095ff0071e11e344c90949f44b/ecs_composex/elbv2/elbv2_ecs.py#L328C16-L328C16

Here in `handle_ping_settings`: 
```
        ("HealthCheckIntervalSeconds", (2, 120)),
        ("HealthCheckTimeoutSeconds", (2, 10)),
```

But based on the docs I would have expected:
```
        ("HealthCheckIntervalSeconds", (5, 300)),
        ("HealthCheckTimeoutSeconds", (2, 120)),
```

https://docs.aws.amazon.com/elasticloadbalancing/latest/application/target-group-health-checks.html
https://docs.aws.amazon.com/elasticloadbalancing/latest/network/target-group-health-checks.html

Hence with the following `healthcheck`:
`healthcheck: 5001:HTTP:2:10:300:30:/:200`

I'm getting during `ecs-compose-x up`:
```
2023-12-30 17:17:29 [   ERROR] Value for HealthCheckIntervalSeconds is not valid. Must be in range of (2, 120)
2023-12-30 17:17:29 [   ERROR] Value for HealthCheckTimeoutSeconds is not valid. Must be in range of (2, 10)
```

Though not really urgent, since it still deploys to AWS fine & AWS does their own validations.

### PR DESCRIPTION
**Pull request by louismollick**
Fixes #732 

I haven't tested this change ; but seems like an expected change given NLB was updated in November 2022: 
https://aws.amazon.com/about-aws/whats-new/2022/11/elastic-load-balancing-capabilities-application-availability/
For example the CDK had to update their validations too:
https://github.com/aws/aws-cdk/issues/26023

New values are as described in the docs:
https://docs.aws.amazon.com/elasticloadbalancing/latest/network/target-group-health-checks.html

### CODE REVIEW

File: ecs_composex/elbv2/elbv2_ecs.py [hunk 1]
```diff
@@ -261,57 +261,6 @@ def set_new_resource_outputs(self, output_definition):
         return value
 
 
-def validate_tcp_health_counts(props):
```
Comment (JohnPreston) [inline ecs_composex/elbv2/elbv2_ecs.py:264]: Why are we removing this function? Is it totally unused and I should have deleted it long time ago?

Comment (louismollick) [reply ecs_composex/elbv2/elbv2_ecs.py:264]: https://github.com/compose-x/ecs_composex/blob/main/ecs_composex/elbv2/elbv2_ecs.py#L312

`validate_tcp_health_counts` is only ever used inside function `fix_nlb_settings` which we're also removing below

Comment (JohnPreston) [reply ecs_composex/elbv2/elbv2_ecs.py:264]: Have you tried then to have `HealthCheckIntervalSeconds` set 6 or 7 then and did it work? (by work, I mean successful deployment to AWS via CFN)
The point of these validations / fix functions is to catch incorrect input and correct it with best effort.

I must have missed that news on the change of health check possible for NLBs/ALBs.

Comment (JohnPreston) [reply ecs_composex/elbv2/elbv2_ecs.py:264]: @louismollick any insignts on this? did you try creating the NLB with invalid interval values for example? if so, what happened?
thank you

**Comment by JohnPreston** [comment]
@louismollick sorry for the "delay" in getting this merged. Was still waiting on your reply. I'd love to merge this so that gets in with the elbv2 lookup together :)

File: ecs_composex/elbv2/elbv2_ecs.py (hunk 1 above)
Comment (louismollick) [reply ecs_composex/elbv2/elbv2_ecs.py:264]: @JohnPreston Hey! Sorry for the radio silence, I got sidetracked

I edited and deployed the nginx example from your repo to show the NLB values: 
https://github.com/compose-x/ecs_composex/compare/main...louismollick:ecs_composex:elbv2ParamsFixDemo 
```
healthcheck: 80:TCP:2:10:300:120
```
^^ here the `HealthCheckIntervalSeconds` is `300` and `HealthCheckTimeoutSeconds` is `120`

I deployed from this branch using `poetry run ecs-compose-x up -f docker-compose.yaml -f aws-compose-x.yaml -n testNLB`

Below are some screenshots from the results in AWS, it even shows these values when you go to edit them in Target Group healthcheck settings:
Screen Shot 2024-01-18 at 11 40 09 PM
Screen Shot 2024-01-18 at 11 40 21 PM

Also attached the template here:
[template.json](https://github.com/compose-x/ecs_composex/files/13984422/template.json)

Comment (JohnPreston) [reply ecs_composex/elbv2/elbv2_ecs.py:264]: Thanks so much. There I thought I was keeping up with all the changes getting the announcement news but there we go, that's some nice changes AWS did!

Thanks @louismollick

**Pull request merged**